### DISCUSSION TITLE
Launch Todoist Minimized on Windows Startup - Mathis Gauthey's Website

### DISCUSSION THREAD

**Discussion by giscus** [mathisgauthey/mathisgauthey.github.io#7] [_]
# Launch Todoist Minimized on Windows Startup - Mathis Gauthey's Website

Todoist doesn't provide a way to launch Todoist on startup, minimized. Here's a script that does it.

https://mathisgauthey.github.io/launch-todoist-minimized-on-windows-startup/

**Comment by belima** [comment]
In case someone else is wondering what to do with it:
1. Download and install AHK v1 (https://www.autohotkey.com/download/ahk-install.exe)
2. Copy and paste Mathis' script in a new .ahk file.
3. Save this .ahk file inside your startup folder.(You can just type <shell:startup> in windows explorer Address Bar)
4. [optional] Test the script by double-clicking the file. Todoist should open and instantly minimize to the taskbar.
5. That's it. Todoist will start and then minimize itself next time you boot up ;)

**Reply by mathisgauthey** [reply]
> In case someone else is wondering what to do with it:
> 1. Download and install AHK v1 (https://www.autohotkey.com/download/ahk-install.exe)
> 2. Copy and paste Mathis' script in a new .ahk file.
> 3. Save this .ahk file inside your startup folder.(You can just type <shell:startup> in windows explorer Address Bar)
> 4. [optional] Test the script by double-clicking the file. Todoist should open and instantly minimize to the taskbar.
> 5. That's it. Todoist will start and then minimize itself next time you boot up ;)

Hey there ! I'll thank you in the name of the future folks that'll find it useful. It was originally just a simple copy pasta for myself on another computer.

If you allow me, I'll add your instructions to the original post and quote you in it. 

Or you can modify it yourself by using the edit button directly inside the page, your call 👌

**Comment by belima** [comment]
Hi!

Absolutely! Go ahead ;) Glad it'll be useful.

Also, thank you for the awesome obsidian content and plugins ;)

Cheers from Brazil!


Em qua., 13 de mar. de 2024 13:34, Mathis Gauthey ***@***.***>
escreveu:
